pre-commit.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
```

Comment:
Hum, I think it'll be just one more place that we have to remember to update if we change python version. Why not just use what's on the venv (taking it's python3 at least)?